docs/app/pages/components/components-sections/dashboard/widgets/dashboard-widgets.component.html

```diff
@@ -0,0 +1,188 @@
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="actions" type="ActionConfig[]" [required]="true">The buttons that will be displayed.
+        Includes <code>label</code> (a string),
+        <code>icon</code> (a ux-icon code), <code>iconTemplate</code> (a TemplateRef) and <code>action</code> (a
+        function).
+    </tr>
+    <tr uxd-api-property name="status" type="ActionStatus" [required]="true">The status displayed by the widget.</tr>
+</uxd-api-properties>
+
+<p>Additionally the actions widget has these outputs available:</p>
+
+<uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="actionSelected" type="string">Emits when the selected value has been changed and saved.
+    </tr>
+</uxd-api-properties>
+
+<h3>Enum Widget</h3>
+
+<p>
+    The <code>ux-dashboard-enum-widget</code> is a select input at its core. The label of the currently selected value
+    is displayed in the dropdown input.
+    The value can be changed by using the dropdown input.
+</p>
+
+<p>The enum widget has the following additional attributes:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="value" type="string" [required]="true"></tr>
+    <tr uxd-api-property name="enums" type="EnumConfig[]" [required]="true"></tr>
+</uxd-api-properties>
+
+<p>Additionally the enum widget has these outputs available:</p>
+
+<uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="valueChange" type="string">Emits when the selected value has been changed and saved.</tr>
+</uxd-api-properties>
+
+<h3>Table Widget</h3>
+
+<p>
+    The <code>ux-dashboard-table-widget</code> is a normal HTML <code>table</code> element which renders a header and a
+    data array into a scrollable table.
+</p>
+
+<p>The table widget has the following additional attributes:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="header" type="any[]">The headers for the table columns.</tr>
+    <tr uxd-api-property name="data" type="any[][]" [required]="true">The data being displayed in the table. Each
+        sub-array should be the same length as the header array.
+    </tr>
+</uxd-api-properties>
+
+<h3>Text Widget</h3>
+
+<p>
+    The <code>ux-dashboard-text-widget</code> displays static text. The button in the bottom right opens a side panel
+    which also fills the whole parent dashboard component and displays the full text.
+    Additionally the side panel can be set to be a text editor as well so that the user can edit the text and save it.
+</p>
+
+<p>It has the following additional attributes:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="text" type="string" [required]="true">The text that is being displayed.</tr>
+    <tr uxd-api-property name="editable" type="boolean">Whether the text can be edited in the side panel.</tr>
+    <tr uxd-api-property name="localizedStrings" type="TextWidgetLocalizedStrings" [required]="true">An object that
```

Comment:
TextWidgetLocalizedStrings is not documented. However since there are only four of them, I would suggest adding each one as a string input instead.